app/include/dt-bindings/zmk/keys.h

```diff
@@ -144,11 +143,24 @@
 #define M_VOLU 0xE9
 #define M_VOLD 0xEA
 
-#define MOD_LCTL (1 << 0x00)
-#define MOD_LSFT (1 << 0x01)
-#define MOD_LALT (1 << 0x02)
-#define MOD_LGUI (1 << 0x03)
-#define MOD_RCTL (1 << 0x04)
-#define MOD_RSFT (1 << 0x05)
-#define MOD_RALT (1 << 0x06)
-#define MOD_RGUI (1 << 0x07)
\ No newline at end of file
+#define MOD_LCTL 0x01
```

Comment:
I'd suggest moving the modifiers codes and macros into their own header - perhaps `modifiers.h`?

For now, `modifiers.h` can be included into `keys.h` until the [single entry point](https://discordapp.com/channels/719497620560543766/719565084208398406/769222960892346438) enhancement for keymaps has been implemented.  The includes to `keys.h` elsewhere in the system (such as in `keycode-state-changed.h`) can then be replaced by `modifiers.h` which is a tighter design.

Reply:
Fully agreed. I'm going to merge this from @okke-formsma , and then follow up with that small refactor.